pkg/api/routes.go

```diff
@@ -60,6 +60,12 @@ func (h *Handler) GetRoutes() []Route {
 			h.CreateImage,
 		},
 		{
+			"DeleteImage",
+			"POST",
```

Comment:
@kokhang There can (and should) be more than one path to creating a Ceph Block image, it depends on the use case.  At the rook API level, it should provide all the necessary tools for image management.  Both the rook volume and the rook-operator can make calls to manage block images.  TPR requests can configure block images, which the rook-operator will then implement using the rook API.  The existence of the TPR path doesn't mean that the volume plugin has to use the TPR path, it can still go directly to the rook API.  We just need to make sure that both paths can co-exist and play nicely together.  The TPR path is for exposing Ceph Block images outside of the cluster, e.g. via iSCSI.